DEFAULT_FASTRTPS_PROFILES.xml

```diff
@@ -1,53 +1,53 @@
 <profiles>
-  <participant profile_name="default_xrce_participant_profile">
-    <rtps>
-      <builtin>
-        <leaseDuration>
-          <durationbyname>INFINITE</durationbyname>
-        </leaseDuration>
-        <domainId>0</domainId>
-      </builtin>
-      <name>DataReader_participant_subscriber</name>
-    </rtps>
-  </participant>
-  <publisher profile_name="default_xrce_publisher_profile">
-    <topic>
-      <kind>WITH_KEY</kind>
-      <name>Square</name>
-      <dataType>ShapeType</dataType>
-      <historyQos>
-        <kind>KEEP_LAST</kind>
-        <depth>5</depth>
-      </historyQos>
-      <durability>
-        <kind>TRANSIENT_LOCAL</kind>
-      </durability>
+    <participant profile_name="default_xrce_participant">
+        <rtps>
+            <builtin>
+                <leaseDuration>
+                    <durationbyname>INFINITE</durationbyname>
+                </leaseDuration>
+                <domainId>0</domainId>
+            </builtin>
+            <name>default_xrce_participant</name>
+        </rtps>
+    </participant>
+    <data_writer profile_name="shapetype_data_writer">
+        <topic>
+            <kind>WITH_KEY</kind>
+            <name>Square</name>
+            <dataType>ShapeType</dataType>
+            <historyQos>
+                <kind>KEEP_LAST</kind>
+                <depth>5</depth>
+            </historyQos>
+            <durability>
+                <kind>TRANSIENT_LOCAL</kind>
+            </durability>
+        </topic>
+    </data_writer>
+    <data_reader profile_name="shapetype_data_reader">
+        <topic>
+            <kind>WITH_KEY</kind>
+            <name>Square</name>
+            <dataType>ShapeType</dataType>
+            <historyQos>
+                <kind>KEEP_LAST</kind>
+                <depth>5</depth>
+            </historyQos>
+            <durability>
+                <kind>TRANSIENT_LOCAL</kind>
+            </durability>
+        </topic>
+    </data_reader>
+    <topic profile_name="shapetype_topic">
+        <kind>WITH_KEY</kind>
+        <name>Square</name>
+        <dataType>ShapeType</dataType>
+        <historyQos>
```

Comment:
Simplify xml. Remove history and durability. uses the default.

Reply:
Done!